scripts/create_ozinfo.sh

```diff
@@ -0,0 +1,27 @@
+# create global_ozinfo file for a given date
```

Comment:
Another shebang needed

Reply:
Generally: These new scripts likely need to go in `ush/` and need the style issues addressed